lms/djangoapps/courseware/views/views.py

```diff
@@ -1230,7 +1230,11 @@ def render_xblock(request, usage_key_string, check_if_enrolled=True):
     Returns an HttpResponse with HTML content for the xBlock with the given usage_key.
     The returned HTML is a chromeless rendering of the xBlock (excluding content of the containing courseware).
     """
-    usage_key = UsageKey.from_string(usage_key_string)
+    try:
```

Comment:
@Ayub-Khan I have not been able to find that method. Could you please send me it's link?